app/javascript/src/controller_branches.js

```diff
@@ -171,6 +177,26 @@ BranchesController.createBranch = function($node) {
   return branch;
 }
 
+BranchesController.addBranchCombinator = function(args) {
+  var branch = args[0];
+  if( !branch.index == 0 ) {
```

Comment:
`if( branch.index != 0 )` is easier to read.

See also the other comment about exposing private for public usage since this value can easily be given a different value elsewhere by (undesirable) intention or accident (e.g. mis-type `if( branch.index = 0 )` and you've got a nice bug on your hands).

Reply:
I have no explanation for that if conditional, not sure why I wrote it like that.

This was the main motivation for the switch from 'private' to 'public' for the index var. But keeping it private and using a getter is how it should be done.